samples/file-encryptor/enclave/openssl_src/keys.cpp

```diff
@@ -0,0 +1,438 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openssl/aes.h>
+#include <openssl/evp.h>
+#include <openssl/rand.h>
+#include <string.h>
+
+#include "aes_context.h" // Must be included before common/encryptor.h
+#include "common/encryptor.h"
+#include "common/trace.h"
+
+#define ENCRYPT_OPERATION true
+#define DECRYPT_OPERATION false
+
+void ecall_dispatcher::dump_data(
+    const char* name,
+    unsigned char* data,
```

Comment:
```suggestion
    const unsigned char* data,
```

Reply:
removed this function - was not being used in the sample.